shenyu-native/shenyu-jvmti/src/main/native/head/jawt.h

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright (c) 1999, 2017, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
```

Comment:
@tuohai666 Hi, I'm not sure if `GNU Lincense` is compatible with `Apache Lincense 2.0`. How should I declare the file header?